website/src/content/docs/guides/getting-started.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Getting Started
+description: A guide to setting up Gemini UI
+---
+
+import { Tabs, TabItem } from "@astrojs/starlight/components";
+
+To install Gemini UI run the following command
+
+<Tabs>
+  <TabItem label="npm">``` npm i @gemini-ui-astro/components ```</TabItem>
+  <TabItem label="Yarn">``` yarn add @gemini-ui-astro/components ```</TabItem>
```

Comment:
```suggestion
  <TabItem label="npm">```sh npm i @gemini-ui-astro/components ```</TabItem>
  <TabItem label="Yarn">```sh yarn add @gemini-ui-astro/components ```</TabItem>
```

Does this work?

Reply:
What were you hoping it would do ?

Reply:
Add a copy button 😁